odl/solvers/advanced/chambolle_pock.py

```diff
@@ -148,13 +148,13 @@ def chambolle_pock_solver(op, x, tau, sigma, proximal_primal, proximal_dual,
     """
     # Forward operator
     if not isinstance(op, Operator):
-        raise TypeError('`op` {} is not an instance of {}'
-                        ''.format(op, Operator))
+        raise TypeError('`op` {!r} is not an `Operator`'
```

Comment:
We usually write `<type> instance`. Debatable for sure, but consistency is almost always good :-)